imports/plugins/core/orders/server/methods/updateHistory.js

```diff
@@ -18,20 +19,35 @@ export default function updateHistory(orderId, event, value) {
   check(event, String);
   check(value, Match.Optional(String));
 
+  const authUserId = Reaction.getUserId();
+
   // REVIEW: For marketplace implementations
   // This should be possible for anyone with permission to act on the order
   if (!Reaction.hasPermission("orders")) {
     throw new ReactionError("access-denied", "Access Denied");
   }
 
-  return Orders.update(orderId, {
+  const result = Orders.update({
+    _id: orderId
+  }, {
     $addToSet: {
       history: {
         event,
         value,
-        userId: Reaction.getUserId(),
+        userId: authUserId,
```

Comment:
Looks fine, just wondering for my own knowledge the reason we are setting this to a `const` instead of directly setting it here?

Reply:
Because it's also passed into `appEvents.emit` below and it's always better to avoid running a function twice.